Unable to build wheel for 0.8.1

## 🐛 Bug

I am unable to create a wheel from the v0.8.1 tag. I already have pytorch 1.7 installed.

## To Reproduce

Steps to reproduce the behavior:

1. `git checkout tags/v0.8.1 -b v0.8.1`
1. `python setup.py clean`
1. `python setup.py bdist_wheel --universal --plat-name=manylinux1_x86_64` or `python setup.py install`

The last few couple lines are:
```
/vision/torchvision/csrc/cpu/video/Video.cpp:337:68: error: converting to ‘std::tuple<at::Tensor, double>’ from initializer list would use explicit constructor ‘constexpr std::tuple<_T1, _T2>::tuple(_U1&&, _U2&&) [with _U1 = at::Tensor&; _U2 = double&; <template-parameter-2-3> = void; _T1 = at::Tensor; _T2 = double]’
   std::tuple<torch::Tensor, double> result = {outFrame, frame_pts_s};
                                                                    ^
error: command 'gcc' failed with exit status 1
```
A full log can be viewed here: [output_bdist_wheel.log](https://github.com/pytorch/vision/files/5456508/output_bdist_wheel.log).

I am able to build/install v0.7.0 and earlier versions, but am unable to build v0.8.0 and v0.8.1.

Any help would be appreciated. Thanks!

## Expected behavior

Creation of a 0.8.1 torchvision binary.

## Environment
```
python -m torch.utils.collect_env
Collecting environment information...
/opt/conda/lib/python3.6/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
PyTorch version: 1.7.0
Is debug build: True
CUDA used to build PyTorch: 10.2
ROCM used to build PyTorch: N/A

OS: Ubuntu 16.04.7 LTS (x86_64)
GCC version: (Ubuntu 5.4.0-6ubuntu1~16.04.12) 5.4.0 20160609
Clang version: Could not collect
CMake version: version 3.5.1

Python version: 3.6 (64-bit runtime)
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.16.4
[pip3] torch==1.7.0
[conda] mkl                       2019.4                      243    anaconda
[conda] mkl-include               2019.4                      243    anaconda
[conda] mkl-service               2.3.0            py36he904b0f_0    anaconda
[conda] mkl_random                1.1.0            py36hd6b4f25_0    anaconda
[conda] numpy                     1.16.4           py36h95a1406_0    conda-forge
[conda] torch                     1.7.0                    pypi_0    pypi
```

## Additional context

<!-- Add any other context about the problem here. -->


cc @bjuncek

--- a/torchvision/csrc/cpu/video/Video.cpp
+++ b/torchvision/csrc/cpu/video/Video.cpp
@@ -334,6 +334,5 @@ std::tuple<torch::Tensor, double> Video::Next() {
     LOG(ERROR) << "Decoder failed with ERROR_CODE " << res;
   }
 
-  std::tuple<torch::Tensor, double> result = {outFrame, frame_pts_s};
-  return result;
+  return std::make_tuple(outFrame, frame_pts_s);
 }

